trustscore/src/main/java/io/coti/trustscore/http/data/BucketEventResponseData.java

```diff
@@ -18,6 +18,7 @@
     private String bucketHash;
     private Date startPeriodTime;
     private double calculatedDelta;
+    @SuppressWarnings("java:S1948")
```

Comment:
why this supress?

Reply:
It's suppressing this issue: 
Fields in a Serializable class must themselves be either Serializable or transient even if the class is never explicitly serialized or deserialized.